Gemfile

```diff
@@ -22,7 +22,9 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem "kaminari"
+gem "bootstrap-sass"
+gem "bootstrap-kaminari-views"
```

Comment:
do you really need these gems?

Reply:
I added these gems for views Magesh. While refreshing to load pages to show some design like "loading content/posts" I have added these gems Magesh. Need to add that Magesh. Or if designs are not needed we can remove these gems Magesh

Reply:
Okay let's have it for now.